interfaces/cython/cantera/reaction.pyx

```diff
@@ -165,7 +241,7 @@ cdef class PlogRate(_ReactionRate):
         """
         def __get__(self):
             rates = []
-            cdef vector[pair[double, CxxArrhenius]] cxxrates = self.rate.rates()
+            cdef vector[pair[double, CxxArrhenius]] cxxrates = self.pr().rates()
```

Comment:
With the factories, some things changed so there would be a need to cast (`self.rate` is now always the base class). Since I'm using the helper functions elsewhere, I'd like to leave this in for consistency.